lib/base.sh

```diff
@@ -163,7 +163,7 @@ vset () {
     if [ -z "$value" ]; then
         value=no
     fi
-    if [ "$quiet" -eq 0 ]; then
+    if [ -z "$quiet" ]; then
```

Comment:
But isn’t `quiet` gone? I though this should be `log_level` too. Looks like it was causing problems in my PR #1775.

Reply:
This is actually a [different `quiet`](https://github.com/hpc/charliecloud/blob/37dd7ea8998c6672f653636f04e0704ca0d5d4a6/lib/base.sh#L147) that is only meaningful in the scope of the function `vset`. I missed this detail in my original PR and changed it to `log_level`, which I imagine caused (or would inevitably cause) some problems.

Reply:
Right, OK. I see the comment disagrees with the variable names, so something needs to be cleaned up still.